### PR TITLE
Newsletter-7: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

**Comment by bittomhan** [comment]
I have picture problems in this file

in local:

```markdown
## Check for broken links
bundle exec htmlproofer --check-html --disable-external --url-ignore '/^\/bin/.*/' ./_site
Running ["LinkCheck", "HtmlCheck", "ScriptCheck", "ImageCheck"] on ["./_site"] on *.html... 


Ran on 1096 files!


- ./_site/zh/newsletters/2018/08/07/index.html
  *  image /img/posts/rbf.png does not have an alt attribute (line 133)
htmlproofer 3.15.0 | Error:  HTML-Proofer found 1 failure!
make: *** [test-after-build] Error 1
Makefile:88: recipe for target 'test-after-build' failed
``` 

in remote enviroment test:

```markdown
4:21:04 PM: "build.command" failed                                        
4:21:04 PM: ────────────────────────────────────────────────────────────────
4:21:04 PM: ​
4:21:04 PM:   Error message
4:21:04 PM:   Command failed with exit code 2: make (https://ntl.fyi/exit-code-2)
4:21:04 PM: ​
4:21:04 PM:   Error location
4:21:04 PM:   In Build command from Netlify app:
4:21:04 PM:   make
4:21:04 PM: ​
4:21:04 PM:   Resolved config
4:21:04 PM:   build:
4:21:04 PM:     command: make
4:21:04 PM:     commandOrigin: ui
4:21:04 PM:     environment:
4:21:04 PM:       - REVIEW_ID
4:21:04 PM:     publish: /opt/build/repo/_site
4:21:04 PM:     publishOrigin: ui
4:21:05 PM: Failed during stage "building site": Build script returned non-zero exit code: 2
4:21:05 PM: Build failed due to a user error: Build script returned non-zero exit code: 2
4:21:05 PM: Failing build: Failed to build site
4:21:05 PM: Finished processing build request in 7m30.057s
``` 

How to debug the image error? What should I do?

File: _posts/zh/newsletters/2018-08-07-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+lang: zh
+---
+本周的新闻简报包括常规的仪表板和行动项目，一个关于通过闪电网络进行通用比特币合约讨论的链接，一个最近宣布的用于增强可扩展性的 BLS 签名库的简要描述，以及来自 Bitcoin Core、LND 和 C-Lightning 项目的一些值得注意的提交。
+
+## 行动项目
+
+- Optech 已开始计划其首次欧洲工作坊，预计将于 11 月在巴黎举行。如果任何认为他们将能够参加的会员公司对讨论的主题感兴趣，请[给 Optech 发送电子邮件][optech email]。关于工作坊的更多信息将在几周内发布。
+
+## 仪表板项目
+
```
Comment (hulatown) [inline _posts/zh/newsletters/2018-08-07-newsletter.md:17]:
```suggestion
{% assign img1_label = "Transactions signaling opt-in RBF, August 2017 - August 2018" %}
```

Comment (hulatown) [reply _posts/zh/newsletters/2018-08-07-newsletter.md:17]: need to add this label to solve the compiling problem

Comment (bitschmidty) [reply _posts/zh/newsletters/2018-08-07-newsletter.md]: Thanks @hulatown !

File: _posts/zh/newsletters/2018-08-07-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  end="f0f5e11b826e020c11c37343bcbaf9725627378b"
+%}
+{% include linkers/github-log.md
+  refname="cl commits"
+  repo="ElementsProject/lightning"
+  start="0b597f671aa31c1c56d32a554fcdf089646fc7c1"
+  end="80a8e57ede82292818032eeb3510da067fddfd5e"
+%}
+
+- **<!--bitcoin-core-13697-->[Bitcoin Core #13697][]**：Pieter Wuille 在 [Newsletter #5][] 中提到的这个 PR 已被合并，它为即将发布的 0.17 RPC `scantxoutset` 添加了[输出脚本描述符][output script descriptors]支持。这些描述符为软件提供了一种全面的方式来描述你想要找到的输出脚本，预计将随时间适应 Bitcoin Core API 的其他部分，如 [importprivkey][rpc importprivkey]、[importaddress][rpc importaddress]、[importpubkey][rpc importpubkey]、[importmulti][rpc importmulti] 和 [importwallet][rpc importwallet]。
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2018-08-07-newsletter.md:61]: In some of the first newsletters we had the items here in bold with ** asterisks, but stopped doing that. I think it makes sense zh formatting matches. Can you see about removing the bold items?

Comment (bitschmidty) [inline _posts/zh/newsletters/2018-08-07-newsletter.md:61]: We should link the newsletter 5 to the zh version

File: _posts/zh/newsletters/2018-08-07-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    ![{{img1_label}}](/img/posts/rbf.png)
+    *{{img1_label}},
+    source: [Optech dashboard][rbf data]*
+
+## 新闻
+
+- **<!--discussion-of-arbitrary-contracts-over-ln-->关于通过 LN 进行任意合约的讨论：**上周 Lightning Network (LN) 开发邮件列表上的一个[帖子][contract thread]描述了在支付通道中执行任意比特币合约的基本原则。与独立的合约相比，该合约解析为 True 以成为有效交易，完全相同的合约被包含在 LN 支付中，并且必须返回 true 以使通道内支付交易有效。任意合约中的所有其他内容以及 LN 支付可以保持不变，具体例外在知识渊博的匿名研究员 ZmnSCPxj 启动的这个帖子中进行了讨论。
+
+- **<!--library-announced-for-bls-signatures-->宣布用于 BLS 签名的库：**著名开发者 Bram Cohen [宣布][bls announce]了一个“初版（但功能完备的）用于基于 [MuSig][] 构造的 [BLS 签名][BLS signatures]库”。这些签名提供了与 [Newsletter #3 特别新闻][#3 schnorr] 中描述的 Schnorr 签名相同的大部分好处，但也允许非交互式签名聚合，这可以通过减少区块链中的签名数据量（可能是非常大的百分比）来提高可扩展性，并通过实施非交互式 CoinJoin 技术（如 [Mimblewimble 论文][Mimblewimble
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2018-08-07-newsletter.md:30]: We should update newsletter 3 to the zh version

**Review by bitschmidty — CHANGES_REQUESTED** [review]

File: _posts/zh/newsletters/2018-08-07-newsletter.md (hunk 2 above)
Comment (bittomhan) [reply _posts/zh/newsletters/2018-08-07-newsletter.md]: I have removed the Bold formatting from this section. Going forward, unless necessary, such as in Issue 10 where a single 0 cannot serve as an anchor link, I will not uniformly apply Bold formatting.

Comment (bittomhan) [reply _posts/zh/newsletters/2018-08-07-newsletter.md]: The Chinese link for Issue 5 has been added. Thank you for the reminder, and I will be mindful of this in the future.

File: _posts/zh/newsletters/2018-08-07-newsletter.md (hunk 3 above)
Comment (bittomhan) [reply _posts/zh/newsletters/2018-08-07-newsletter.md:30]: The Chinese link for Issue 3 has been added as well. Thank you for the reminder, and I will ensure to pay attention to this moving forward.

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @bittomhan

**Pull request merged**